internal/distro/fedora31/distro_test.go

```diff
@@ -1,31 +1,11 @@
 package fedora31_test
 
 import (
-	"reflect"
 	"testing"
 
 	"github.com/osbuild/osbuild-composer/internal/distro/fedora31"
 )
 
-func TestListOutputFormats(t *testing.T) {
```

Comment:
This should still be possible, just more complicated, no? For example a test case testing a matrix of arches and image types.

Reply:
Yeah, totally. Also need tests for the other newly introduced functions. I'll post a follow up to add this.